modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/average_finder.hpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2021-2023 Intel Corporation
```

Comment:
Could you please split PR to 2 PRs: I would like to discuss automatic threshold calculation separately. Other fixes should be merged asap.

Reply:
Done.
The 2nd PR is https://github.com/openvinotoolkit/openvino_contrib/pull/677